tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -5481,6 +5667,80 @@ bool CudnnSupport::GetConvolveAlgorithms(
   return true;
 }
 
+tsl::StatusOr<std::unique_ptr<const dnn::FusedMHASimpleRunner>>
+CudnnSupport::FusedMHASimpleRunnerFromDesc(
+    Stream* stream, const dnn::AlgorithmDesc& algorithm_desc,
+    dnn::FusedMHAKind kind,
+    const dnn::MatmulTensorDescriptor& bmm1_lhs_descriptor,
+    const dnn::MatmulTensorDescriptor& bmm1_rhs_descriptor,
+    const dnn::MatmulTensorDescriptor& bmm2_rhs_descriptor,
+    const dnn::MatmulTensorDescriptor& intermediate_bmm2_lhs_descriptor,
+    const dnn::TensorDescriptor& output_descriptor,
+    std::optional<double> dropout_rate) {
+#if CUDNN_VERSION >= 8100 && TF_ENABLE_CUDNN_FRONTEND
+  auto cudnn = cudnn_->GetHandle(parent_, stream);
+  TF_ASSIGN_OR_RETURN(auto op_graph,
+                      GetCudnnFusedMHASimpleOperationGraph(
+                          kind, bmm1_lhs_descriptor, bmm1_rhs_descriptor,
+                          bmm2_rhs_descriptor, intermediate_bmm2_lhs_descriptor,
+                          output_descriptor, dropout_rate, cudnn));
+  // The function  GetExecPlanFromHeuristics uses cudnn_frontend::cudnnException
+  // which is currently not
+  // recommended for use by Google. Hence commenting out the call to this
+  // function until this gets resolved.
+  // TF_ASSIGN_OR_RETURN(auto execution_plan,
```

Comment:
We do not check in commented out code. If you need exception support, could you add wrappers which will be compiled with their own rules, and convert exceptions to Status?

Reply:
We plan to add some support for wrappers for `cudnn_frontend::cudnnException`. However, that doesn't seem to a high priority right now since we have a workaround to use knob ids n values. Removing the commented code.